gdcdictionary/schemas/germline_mutation_calling_workflow.yaml

```diff
@@ -21,18 +21,26 @@ systemProperties:
   - state
 
 links:
-  - name: aligned_reads_files
-    backref: germline_mutation_calling_workflows
-    label: performed_on
-    target_type: aligned_reads
-    multiplicity: many_to_many
+  - exclusive: true
     required: true
+    subgroup:
+      - name: aligned_reads_files
+        backref: germline_mutation_calling_workflows
+        label: performed_on
+        target_type: aligned_reads
+        multiplicity: many_to_many
+        required: true
+      - name: germline_variation_catalogs
+        backref: germline_mutation_calling_workflows
+        label: performed_on
+        target_type: germline_variation_catalog
+        multiplicity: many_to_many
+        required: true
```

Comment:
Wait, is this allowed? Aren't you requiring both links exist? That would conflict with the `exclusive: true` wouldn't it? 😕

Reply:
Yup - snuck in there with some copy/paste I think - fixed